packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -177,8 +189,8 @@ module.exports = {
 
       // First, run the linter.
       // It's important to do this before Babel processes the JS.
-      {
-        test: /\.(js|mjs|jsx)$/,
+      !paths.useTSLint && {
```

Comment:
I think this is not cool since disabling ESLint completely when you use TSLint is wrong. For example, I have a big repo where there is JS and TS/TSX files are alongside, so I need to have both ESLint for JS and TSLint for TS/TSX files.
This change will break such interop.

Reply:
What's your suggestion to handle this case and also the case where the person wants to disable eslint and use only tslint?

Reply:
If you have no `js|jsx` files in your project ESLint run will take less than a second I think. So I don't think it's a good idea to switch ESLint at all. 
TSLint, yeah, should switch on/off on `tsconfig.json` appearance as it works now, it's fine. 

But ESLint should run against the `js|jsx` and that is it. Maybe at some point thanks to Babel understanding TS now, ESLint rules will start to support TS as a first class citizen so we can get rid of TSLint, but not today.

Reply:
Yes, please remove this. ESLint should always run.